kong/clustering/services/sync/rpc.lua

```diff
@@ -177,7 +177,16 @@ local function is_rpc_ready()
 end
 
 
-local function do_sync()
+local do_sync
+
+
+local function error_handler(err)
```

Comment:
Since we have removed some code in this function, I don't think it is necessary now (only two lines).,